### DISCUSSION TITLE
Problem mit Server Infos nach Update auf 1.6.1

### DISCUSSION THREAD

**Discussion by wueste87** [Part-DB/Part-DB-server#344] [General]
Hallo zusammen,

ich habe gestern das Update auf Version 1.6.1 gemacht.
Strikt nach der Anleitung im Wiki.
Seitdem kann ich nicht mehr die Seite  "Tools -> System ->Server Infos" aufrufen.
Fehler 500 - Internal Server Error.
Alle anderen Bereiche von Part-DB sind erreichbar und laufen fehlerfrei.
ich kann ohne Probleme neue Bauteile anlegen u.s.w.

Im prod.log sind einige Fehler enthalten und ich habe dazu auch schon Google bemüht, aber bisher konnte ich das Problem nicht eigenständig lösen.

Das Log schaut wie folgt aus:

`{"message":"Matched route \"tools_server_infos\".","context":{"route":"tools_server_infos","route_parameters":{"_route":"tools_server_infos","_controller":"App\\Controller\\ToolsController::systemInfos","_locale":"de"},"request_uri":"http://192.168.187.99/index.php/de/tools/server_infos","method":"GET"},"level":200,"level_name":"INFO","channel":"request","datetime":"2023-08-09T18:15:09.954696+00:00","extra":{"token":null,"url":"/index.php/de/tools/server_infos","ip":"192.168.187.49","http_method":"GET","server":"192.168.187.99","referrer":null}}
{"message":"Checking for authenticator support.","context":{"firewall_name":"main","authenticators":4},"level":100,"level_name":"DEBUG","channel":"security","datetime":"2023-08-09T18:15:09.964388+00:00","extra":{"token":null,"url":"/index.php/de/tools/server_infos","ip":"192.168.187.49","http_method":"GET","server":"192.168.187.99","referrer":null}}
{"message":"Checking support on authenticator.","context":{"firewall_name":"main","authenticator":"Scheb\\TwoFactorBundle\\Security\\Http\\Authenticator\\TwoFactorAuthenticator"},"level":100,"level_name":"DEBUG","channel":"security","datetime":"2023-08-09T18:15:09.964482+00:00","extra":{"token":null,"url":"/index.php/de/tools/server_infos","ip":"192.168.187.49","http_method":"GET","server":"192.168.187.99","referrer":null}}
{"message":"Authenticator does not support the request.","context":{"firewall_name":"main","authenticator":"Scheb\\TwoFactorBundle\\Security\\Http\\Authenticator\\TwoFactorAuthenticator"},"level":100,"level_name":"DEBUG","channel":"security","datetime":"2023-08-09T18:15:09.964559+00:00","extra":{"token":null,"url":"/index.php/de/tools/server_infos","ip":"192.168.187.49","http_method":"GET","server":"192.168.187.99","referrer":null}}
{"message":"Checking support on authenticator.","context":{"firewall_name":"main","authenticator":"Nbgrp\\OneloginSamlBundle\\Security\\Http\\Authenticator\\SamlAuthenticator"},"level":100,"level_name":"DEBUG","channel":"security","datetime":"2023-08-09T18:15:09.964614+00:00","extra":{"token":null,"url":"/index.php/de/tools/server_infos","ip":"192.168.187.49","http_method":"GET","server":"192.168.187.99","referrer":null}}
{"message":"Authenticator does not support the request.","context":{"firewall_name":"main","authenticator":"Nbgrp\\OneloginSamlBundle\\Security\\Http\\Authenticator\\SamlAuthenticator"},"level":100,"level_name":"DEBUG","channel":"security","datetime":"2023-08-09T18:15:09.964670+00:00","extra":{"token":null,"url":"/index.php/de/tools/server_infos","ip":"192.168.187.49","http_method":"GET","server":"192.168.187.99","referrer":null}}
{"message":"Checking support on authenticator.","context":{"firewall_name":"main","authenticator":"Symfony\\Component\\Security\\Http\\Authenticator\\FormLoginAuthenticator"},"level":100,"level_name":"DEBUG","channel":"security","datetime":"2023-08-09T18:15:09.964720+00:00","extra":{"token":null,"url":"/index.php/de/tools/server_infos","ip":"192.168.187.49","http_method":"GET","server":"192.168.187.99","referrer":null}}
{"message":"Authenticator does not support the request.","context":{"firewall_name":"main","authenticator":"Symfony\\Component\\Security\\Http\\Authenticator\\FormLoginAuthenticator"},"level":100,"level_name":"DEBUG","channel":"security","datetime":"2023-08-09T18:15:09.964776+00:00","extra":{"token":null,"url":"/index.php/de/tools/server_infos","ip":"192.168.187.49","http_method":"GET","server":"192.168.187.99","referrer":null}}
{"message":"Checking support on authenticator.","context":{"firewall_name":"main","authenticator":"Symfony\\Component\\Security\\Http\\Authenticator\\RememberMeAuthenticator"},"level":100,"level_name":"DEBUG","channel":"security","datetime":"2023-08-09T18:15:09.964825+00:00","extra":{"token":null,"url":"/index.php/de/tools/server_infos","ip":"192.168.187.49","http_method":"GET","server":"192.168.187.99","referrer":null}}
{"message":"Authenticator does not support the request.","context":{"firewall_name":"main","authenticator":"Symfony\\Component\\Security\\Http\\Authenticator\\RememberMeAuthenticator"},"level":100,"level_name":"DEBUG","channel":"security","datetime":"2023-08-09T18:15:09.964950+00:00","extra":{"token":null,"url":"/index.php/de/tools/server_infos","ip":"192.168.187.49","http_method":"GET","server":"192.168.187.99","referrer":null}}
{"message":"Read existing security token from the session.","context":{"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"},"level":100,"level_name":"DEBUG","channel":"security","datetime":"2023-08-09T18:15:09.970399+00:00","extra":{"token":null,"url":"/index.php/de/tools/server_infos","ip":"192.168.187.49","http_method":"GET","server":"192.168.187.99","referrer":null}}
{"message":"User Deprecated: Using XML mapping driver with XSD validation disabled is deprecated and will not be supported in Doctrine ORM 3.0. (XmlDriver.php:60 called by App_KernelProdContainer.php:296, https://github.com/doctrine/orm/pull/6728, package doctrine/orm)","context":{"exception":{"class":"ErrorException","message":"User Deprecated: Using XML mapping driver with XSD validation disabled is deprecated and will not be supported in Doctrine ORM 3.0. (XmlDriver.php:60 called by App_KernelProdContainer.php:296, https://github.com/doctrine/orm/pull/6728, package doctrine/orm)","code":0,"file":"/var/www/Part-DB-symfony/vendor/doctrine/deprecations/lib/Doctrine/Deprecations/Deprecation.php:209"}},"level":200,"level_name":"INFO","channel":"deprecation","datetime":"2023-08-09T18:15:09.972086+00:00","extra":{"token":null,"url":"/index.php/de/tools/server_infos","ip":"192.168.187.49","http_method":"GET","server":"192.168.187.99","referrer":null}}
{"message":"User Deprecated: Version detection logic for MySQL will change in DBAL 4. Please specify the version as the server reports it, e.g. \"10.9.3-MariaDB\" instead of \"mariadb-10.9\". (AbstractMySQLDriver.php:125 called by AbstractMySQLDriver.php:37, https://github.com/doctrine/dbal/pull/5779, package doctrine/orm)","context":{"exception":{"class":"ErrorException","message":"User Deprecated: Version detection logic for MySQL will change in DBAL 4. Please specify the version as the server reports it, e.g. \"10.9.3-MariaDB\" instead of \"mariadb-10.9\". (AbstractMySQLDriver.php:125 called by AbstractMySQLDriver.php:37, https://github.com/doctrine/dbal/pull/5779, package doctrine/orm)","code":0,"file":"/var/www/Part-DB-symfony/vendor/doctrine/deprecations/lib/Doctrine/Deprecations/Deprecation.php:209"}},"level":200,"level_name":"INFO","channel":"deprecation","datetime":"2023-08-09T18:15:09.982060+00:00","extra":{"token":null,"url":"/index.php/de/tools/server_infos","ip":"192.168.187.49","http_method":"GET","server":"192.168.187.99","referrer":null}}
{"message":"User Deprecated: Since symfony/doctrine-bridge 6.3: Registering \"App\\EventSubscriber\\LogSystem\\EventLoggerSubscriber\" as a Doctrine subscriber is deprecated. Register it as a listener instead, using e.g. the #[AsDoctrineListener] attribute.","context":{"exception":{"class":"ErrorException","message":"User Deprecated: Since symfony/doctrine-bridge 6.3: Registering \"App\\EventSubscriber\\LogSystem\\EventLoggerSubscriber\" as a Doctrine subscriber is deprecated. Register it as a listener instead, using e.g. the #[AsDoctrineListener] attribute.","code":0,"file":"/var/www/Part-DB-symfony/vendor/symfony/doctrine-bridge/ContainerAwareEventManager.php:192"}},"level":200,"level_name":"INFO","channel":"deprecation","datetime":"2023-08-09T18:15:09.992932+00:00","extra":{"token":null,"url":"/index.php/de/tools/server_infos","ip":"192.168.187.49","http_method":"GET","server":"192.168.187.99","referrer":null}}
{"message":"User Deprecated: Since symfony/doctrine-bridge 6.3: Registering \"App\\EventSubscriber\\LogSystem\\LogDBMigrationSubscriber\" as a Doctrine subscriber is deprecated. Register it as a listener instead, using e.g. the #[AsDoctrineListener] attribute.","context":{"exception":{"class":"ErrorException","message":"User Deprecated: Since symfony/doctrine-bridge 6.3: Registering \"App\\EventSubscriber\\LogSystem\\LogDBMigrationSubscriber\" as a Doctrine subscriber is deprecated. Register it as a listener instead, using e.g. the #[AsDoctrineListener] attribute.","code":0,"file":"/var/www/Part-DB-symfony/vendor/symfony/doctrine-bridge/ContainerAwareEventManager.php:192"}},"level":200,"level_name":"INFO","channel":"deprecation","datetime":"2023-08-09T18:15:09.994499+00:00","extra":{"token":null,"url":"/index.php/de/tools/server_infos","ip":"192.168.187.49","http_method":"GET","server":"192.168.187.99","referrer":null}}
{"message":"User Deprecated: Subscribing to postConnect events is deprecated. Implement a middleware instead. (Connection.php:385 called by Connection.php:1652, https://github.com/doctrine/dbal/issues/5784, package doctrine/dbal)","context":{"exception":{"class":"ErrorException","message":"User Deprecated: Subscribing to postConnect events is deprecated. Implement a middleware instead. (Connection.php:385 called by Connection.php:1652, https://github.com/doctrine/dbal/issues/5784, package doctrine/dbal)","code":0,"file":"/var/www/Part-DB-symfony/vendor/doctrine/deprecations/lib/Doctrine/Deprecations/Deprecation.php:209"}},"level":200,"level_name":"INFO","channel":"deprecation","datetime":"2023-08-09T18:15:09.995139+00:00","extra":{"token":null,"url":"/index.php/de/tools/server_infos","ip":"192.168.187.49","http_method":"GET","server":"192.168.187.99","referrer":null}}
{"message":"User was reloaded from a user provider.","context":{"provider":"Symfony\\Bridge\\Doctrine\\Security\\User\\EntityUserProvider","username":"admin"},"level":100,"level_name":"DEBUG","channel":"security","datetime":"2023-08-09T18:15:10.000056+00:00","extra":{"token":null,"url":"/index.php/de/tools/server_infos","ip":"192.168.187.49","http_method":"GET","server":"192.168.187.99","referrer":null}}
{"message":"Uncaught PHP Exception Twig\\Error\\LoaderError: \"Unable to find template \"components/new_version.macro.html.twig\" (looked into: /var/www/Part-DB-symfony/templates, /var/www/Part-DB-symfony/vendor/symfony/twig-bridge/Resources/views/Form) in \"tools/server_infos/server_infos.html.twig\" at line 2.\" at /var/www/Part-DB-symfony/vendor/twig/twig/src/Loader/FilesystemLoader.php line 227","context":{"exception":{"class":"Twig\\Error\\LoaderError","message":"Unable to find template \"components/new_version.macro.html.twig\" (looked into: /var/www/Part-DB-symfony/templates, /var/www/Part-DB-symfony/vendor/symfony/twig-bridge/Resources/views/Form) in \"tools/server_infos/server_infos.html.twig\" at line 2.","code":0,"file":"/var/www/Part-DB-symfony/vendor/twig/twig/src/Loader/FilesystemLoader.php:227"}},"level":500,"level_name":"CRITICAL","channel":"request","datetime":"2023-08-09T18:15:10.040577+00:00","extra":{"token":{"authenticated":true,"roles":["ROLE_USER"],"user_identifier":"admin"},"url":"/index.php/de/tools/server_infos","ip":"192.168.187.49","http_method":"GET","server":"192.168.187.99","referrer":null}}
{"message":"Stored the security token in the session.","context":{"key":"_security_main"},"level":100,"level_name":"DEBUG","channel":"security","datetime":"2023-08-09T18:15:10.050423+00:00","extra":{"token":{"authenticated":true,"roles":["ROLE_USER"],"user_identifier":"admin"},"url":"/index.php/de/tools/server_infos","ip":"192.168.187.49","http_method":"GET","server":"192.168.187.99","referrer":null}}`

Jemand eine Idee was die Ursache für mein Problem sein könnte?

Part-DB läuft bei mir auf einem Pi4 mit Bullseye.
PHP 8.2
Apache2
Maria-DB 10.9.3

Vorab schon Mal vielen Dank

**Comment by jbtronics** [comment]
Bist du sicher dass du die 1.6.1 installiert hast? Er findet eine Datei nicht, die ist Teil des Update-Benachrichtigungssystems, das wird es aber erst ab der 1.7.0 geben.

Ansonsten probiere mal den cache zu löschen mit `sudo -u www-data php bin/console cache:clear`. Möglicherweise stehen im cache noch falsche einträge.

**Reply by wueste87** [reply]
Hey Jan, danke für den Hinweis. 
Ich war tatsächlich auf dem falschen Commit.
Jetzt klappt alles.

Eine Sache die mir noch aufgefallen ist, ist das die Vorschaubilder sehr klein sind, sobald ein Bauteil dabei ist, wo die Beschreibung etwas länger ausfällt.

<img width="1039" alt="Screenshot 2023-08-12 003846" src="https://github.com/Part-DB/Part-DB-server/assets/100490800/31e5bab6-9c5f-402c-a74f-682d526f69dd">

PS: Ein großes Danke für dieses ausgezeichnete Projekt.
Echt genial was Du/Ihr hier auf die Beine gestellt habt.
